src/Api/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -312,31 +311,6 @@ await _providerBillingService.ScaleSeats(
         await _organizationService.DeleteAsync(organization);
     }
 
-    [HttpPost("{id}/import")]
-    public async Task Import(string id, [FromBody] ImportOrganizationUsersRequestModel model)
```

Comment:
You can also delete `ImportOrganizationUsersRequestModel`, which doesn't appear to be used anywhere else. (The public api has its own model.)